src/include/kvs/restapi.h

```diff
@@ -58,6 +61,9 @@ typedef struct
     char *pcStreamName;
     FragmentTimecodeType_t xTimecodeType;
     uint64_t uProducerStartTimestampMs;
+
+    unsigned int uRecvTimeoutMs;
+    unsigned int uSendTimeoutMs;

```

Comment:
Quick question, why we use `unsigned int` here instead of `uint64_t` or `uint32_t`like previous variables?

Reply:
My first thought is we might use other network I/O like openssl, so it'd better to use general data type for those exposed transport options.